python/src/TaylorExpansionMoments_doc.i.in

```diff
@@ -12,87 +11,13 @@ limitStateVariable : :class:`~openturns.RandomVector`
 Notes
 -----
 Assuming that :math:`\uX` has its two first moments finite and that :math:`h` is sufficiently regular,
-the Taylor expansion method is used to approximate the mean and variance of a random variable :math:`\uY = h(\uX)`
-with respect to the mean and variance of the input random vector :math:`\uX`.
+the Taylor expansion method is used to approximate the mean and variance of the random variable :math:`\uY = h(\uX)`.
 
-We note:
+Refer to :ref:`taylor_expansion_moments` to get details on the expressions of the approximations:
```

Comment:
This is one of the excellent ideas of this PR: pointing to the theory help page, without duplicating the content. Hence, removing the equations from the API help seems to be a very good idea from my point of view.

Reply:
yes! I generalized it with the files QuadraticTaylor_doc.i.in and LinearTaylor_doc.i.in